wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecord.g4

```diff
@@ -104,6 +105,10 @@ xmldefaultval:
     'XMLDEFAULT' OPAREN STRING CPAREN
 ;
 
+setdefaultval:
+    ('DEFAULT' OPAREN OSQUARE STRING CSQUARE CPAREN | 'DEFAULT' OPAREN SETSTRING CPAREN)
```

Comment:
I think

`
(
'DEFAULT' OPAREN OSQUARE value_list CSQUARE CPAREN |
'DEFAULT' OPAREN OSQUARE CSQUARE CPAREN
)
`
would be a more accurate rule.  It would allow for lists of integers.  The second production is for the empty list (which you will want to implement differently).  Note you could also have the keyword ALL, but I suspect that is very unlikely and I think you are only aiming to cover the common cases.